src/packages/@ncigdc/modern_components/environment.js

```diff
@@ -20,9 +21,12 @@ function fetchQuery(operation, variables, cacheConfig, uploadables) {
   });
 
   const hash = md5(body);
-
   const [componentName] = operation.name.split('_relayQuery');
 
+  if (simpleCache[hash]) {
+    return Promise.resolve(simpleCache[hash]);
+  }
+
   setTimeout(() => reduxStore.dispatch(setLoader(componentName)));
 
   return fetch(
```

Comment:
well actually I think it might not be better to do that (unless I'm missing something). the args for fetch might change, but the if the hash is the same then the result should be the same